src/app/pages/resources/overview/overview.jsx

```diff
@@ -202,31 +210,26 @@ function ChplResourcesOverview() {
             <li>Microsoft Edge (two most recent major versions)</li>
             <li>Mozilla Firefox (latest version)</li>
           </ul>
-          <InternalScrollButton
-            name="Back to the top"
-            id="main-content"
-            analytics={{ event: 'Jump to top of Overview', category: 'Navigation' }}
-          />
           <Divider />
           <span className="anchor-element">
             <a id="usingTheChplWebsite" className="page-anchor" />
           </span>
-          <Typography variant="h2">
+          <Typography gutterBottom variant="h2">
             Using the CHPL Website
           </Typography>
-          <Typography variant="h3">
+          <Typography gutterBottom variant="h5">
```

Comment:
Why is this jumping from h2 to h5? We're skipping h3 and h4. Same with the other h5s

Reply:
So i did this purely for styling purpose but not that I'm realizing its bad practice for semantics. 

I think i need to revisit our theme file and take out any weight from h2-h6 so we don't have odd bolding appear.
Or I can switch the "h5" to "subtitles" so it reads better